### PR TITLE
Add QEII Memorial Bank Holiday

### PR DESCRIPTION
**Pull request by hlascelles**
With tests.

### CODE REVIEW

File: gb.yaml [hunk 1]
```diff
@@ -507,6 +513,21 @@ tests:
       regions: ["gb"]
     expect:
       name: "Platinum Jubilee"
+  - given:
+      date: '2022-09-18'
```
Comment (Jonic) [inline gb.yaml:517]: Should this be the 19th instead?
```suggestion
      date: '2022-09-19'
```

Comment (hlascelles) [reply gb.yaml]: Yes, you are right... Fixed!

Actually, that means (checks...) ... that `make test` doesn't run the `expect` sections?

File: gb.yaml [hunk 2]
```diff
@@ -108,6 +108,12 @@ months:
     regions: [gb_eng, gb_wls, gb_eaw, gb_nir, je, gb_jsy, gg, gb_gsy]
     week: -1
     wday: 1
+  9:
+  - name: Queen Elizabeth II Memorial Bank Holiday
```
Comment (frankieroberto) [inline gb.yaml:112]:
```suggestion
  - name: Bank Holiday for the State Funeral of Queen Elizabeth II
```

Comment (frankieroberto) [reply gb.yaml:112]: May as well use the official wording from https://www.gov.uk/bank-holidays ?

**Review by frankieroberto — CHANGES_REQUESTED** [review]

File: gb.yaml (hunk 1 above)
Comment (frankieroberto) [reply gb.yaml]: No, the tests are run within a separate repo (https://github.com/holidays/holidays ) which imports the definitions and the tests.

**Review by frankieroberto — APPROVED** [review]

**Comment by frankieroberto** [comment]
@ttwo32 could we get this merged asap? 🙏  The extra bank holiday was announced only a week before the event, and so quite a few downstream services need this update as soon as possible.

**Comment by WorkSutton** [comment]
Is there any chance this will be merged in today? My project uses the holidays gem and we have to decide by COP (close of play) today if we are to add this ourselves, in the likelihood that this is not merged today. 
Apologies for the pressure, I would merge if I had capabilities, but for me its `merging is blocked` as I am not a committer/maintainer on the project! 
Thanks in advance

**Comment by ppeble** [comment]
Yes, I will take a look at it this morning. I apologize for the silence, to be quite honest my personal life has taken up almost all of my attention for the past few months (for good reasons, my family is expecting another child in less than a month!). I understand the necessity of the speed of this so I will look before I start work.

**Review by ppeble — APPROVED** [review]

**Comment by frankieroberto** [comment]
> Yes, I will take a look at it this morning. I apologize for the silence, to be quite honest my personal life has taken up almost all of my attention for the past few months (for good reasons, my family is expecting another child in less than a month!). I understand the necessity of the speed of this so I will look before I start work.

Thank you (and congratulations)! Usually we get a bit more notice for a bank holiday...

**Comment by ppeble** [comment]
Okay! I released 8.6.0 of the gem with all the latest definitions. Please give it a shot and if you notice any issues open something here and I will do my best to react ASAP.

**Comment by WorkSutton** [comment]
@ppeble a big thank you and congratulations on your good news. Code delivery is so much simpler! 😄

**Pull request merged**